tests/copyright_check/copyright_validator.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from __future__ import print_function
+
+import difflib
+
+import os
+import re
+import sys
+import pathlib
+import config
+
+print_logs = config.print_logs
+
+
+def fetch_samplefile():
+    """
+    Method to fetch sample copyright file
+    :return sample
+    """
+    sample = dict()
```

Comment:
nit. ditto: use literals

```suggestion
    """
    Method to fetch sample copyright file
    :return sample
    """
    sample = {}
```